README.md

```diff
@@ -83,6 +90,7 @@ These are narrowly-scoped, little JS "apps" deployed through IPFS.
 * [ipfs-linux-service](https://github.com/dylanPowers/ipfs-linux-service) - IPFS Linux Init Daemon
 * [ipfs-paste](https://github.com/jbenet/ipfs-paste) - Paste stdin and clipboard to IPFS
 * [ipfs-screencap](https://github.com/jbenet/ipfs-screencap) - Capture screenshots, publish them to IPFS, and copy the link to the clipboard.
+* [ipfs-screenshot](https://github.com/uptownhr/ipfs-screenshot) - Like Cloudapp using IPFS. Screenshot publisher by selected area. (planning video/gif recorder)
```

Comment:
Seems the repository where this PR was created from, has been deleted. Github shows `unknown repository` so not sure. Could you create a new PR instead?